src/containers/myHouse/addition/ui.tsx

```
@@ -1,11 +1,12 @@
 "use client"
-import React, {useRef, useState} from 'react';
+import React, { useRef, useState } from 'react';
 import * as S from './style';
 import Circle from '@/components/ui/button/circle';
 import Square from '@/components/ui/button/square';
-import {DEFAULT_CONTRACT_OPTIONS, FACILITY_CATEGORIES} from "./data"
-import {useRouter} from 'next/navigation';
-import {useAlertStore} from "@/store/alert";
+import { DEFAULT_CONTRACT_OPTIONS, FACILITY_CATEGORIES } from "./data"
+import { useRouter } from 'next/navigation';
```

ESLint

[error] 'useRouter' is defined but never used.

(@typescript-eslint/no-unused-vars)